website/auth_pages.py

```diff
@@ -168,8 +177,7 @@ def signup(self):
         if not is_testing_request(request) and "subscribe" in body:
             # If we have a Mailchimp API key, we use it to add the subscriber through the API
             if MAILCHIMP_API_URL:
-                role = "teacher" if "is_teacher" in body else "student"
-                mailchimp_subscribe_user(user["email"], body["country"], role)
+                mailchimp_subscribe_user(user["email"], body["country"], "teacher")
```

Comment:
As we are providing a hard-coded string here to the function, maybe remove it all together and make sure the function always handles the request as a teacher?

Reply:
Applied changes!